### ISSUE TITLE
Notes from Tablet are not synchronized to Windows

### ISSUE THREAD

**Issue by dbielz** [adil192/saber#663]
### Describe the bug

If you create a note on a tablet (Samsung Galaxy-Tab S8 in my case) and edit it with the S-pen (active pen), then the note seems to be synced to the server, but stays invisible on the windows instance after that one is synced too. If you type in some text with the keypad and delete the drawings from the pen, the whole note becomes visible in the windows instance as well

### To reproduce

1. open Saber on the tablet. 
2. create a new or modify an existing note with the s-pen.
3. sync to the nextcloud server
4. open Saber on windows desktop
5. sync and see

### Expected behavior

I would expect to see the modified or created note in windows as well. If I create or modify a note on my desktop even with mouse drawing, it is synced and visible on the tablet.

### Saber version

0.12.7 Google Play (12070), the one before as well

### Device

- Device: Dell Inspiron, Galaxy Tab S8
- OS: Windows 11, Android 13

**Comment by sirdrakeistda** [comment]
I had a similar issue. I fixed it by naming the note. Maybe a unnamed note causes some sync conflicts.
actual name for unnamed notes:
date-unnamed
maybe it would be helpful to name like this:
date-time-unnamed
and maybe it would be good to catch sync conflicts at the moment it seems like these conflicts are not handled correctly

**Comment by dbielz** [comment]
Sorry, this did not help. I do not have many notes, since I did not pass over the state of testing. I even deleted all notes, created one new on the tablet with text and one with hand drawing. The text note is synced, the hand note not.

Anyway, the last changes in 0.14.2, enable ads in Saber, rendered the app unusable for me. I do not install any app using ad frameworks. I removed every instance on all of my devices. Sorry for that and thanks for an interesting experiment.
